documents/JupyterHub.md

```diff
@@ -0,0 +1,104 @@
+# Plan for NSLS-II JupyterHub at SDCC
+
+## Goals
+- Make it easy for BL staff to deploy new environments+kernels and make them
+  available to their users. It currently possible to make custom environments
+  for personal use, but the only way to _share_ them is to say, "Run these conda
+  and ipykernel commands," which is not streamlined enough for general users.
+- Provide long-term (~3 year) support for kernels. An old notebook should be able to start
+  its old kernel, so that users can, for example, update an old plot without
+  dealing with any API changes in the related libraries.
+- Make it possible for users to switch between beamlines easily and have
+  notebooks from multiple beamlines open at once.
+- Avoid overwhelming users with a flat list of many kernels.
+
+## Deployment and Computational Resources
+
+### Short term
+
+We will be using the shared "HTC" JupyterHub provided by SDCC. They will add one
```

Comment:
Does "HTC" mean HTCondor here?

Reply:
In practice 'yes', but this term was relating to just "High Throughput Computing" rather than worrying about the technology used for it.

Reply:
Sounds good.